plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -151,6 +155,22 @@ public void testInsertWriteBulkinessWithTimestamps(String timestampType)
         }
     }
 
+    @Test(dataProvider = "testTableNameDataProvider")
+    public void testCreateTableWithTrickyName(String tableName)
```

Comment:
nit: to match existing tests having similar name (makes finding related tests easier).

```suggestion
    public void testCreateTableWithSpecialCharacterName(String tableName)
```

Reply:
Move to BaseConnectorTest (either in this PR or as follow-up - since I expect this to fail in lots of places (ClickHouse, Hive/Iceberg/Delta etc.)

Reply:
Yep let's do it in follow up PR